keps/sig-windows/1301-windows-runtime-class/kep.yaml

```diff
@@ -18,3 +18,6 @@ last-updated: 2019-10-15
 status: implementable
 see-also:
   - "/keps/sig-windows/20190424-windows-cri-containerd.md"
+
+latest-milestone: "1.0"
+stage: "alpha"
```

Comment:
Since we have validation on values and I was too lazy to add "unknowns" I set the defaults to 1.0/alpha for everything that was marked implementable.

Reply:
Instead of "0.0", can we maybe set "0.0" - to make it more clear that it's a marker?